### PR TITLE
Fix default collapse mode-value. Fix collapse unordered animation.

### LINKED ISSUE #1063: Collapse when mounted with v-model true is still closed
Go to `VaCollapse.demo.vue` and set

```
collapseValue: true,
```

Collapses are closed even though icon points upward. They should be open.
image

### PR DESCRIPTION
**Pull request by m0ksem**
## Description
closes #1063

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-collapse/VaCollapse.vue [hunk 1]
```diff
@@ -207,14 +206,18 @@ export default class VaCollapse extends mixins(
   mounted () {
     this.getHeight()
 
+    this.setCollapseParams()
+
     this.mutationObserver = new MutationObserver(() => {
-      this.setCollapseParams()
+      setTimeout(() => this.setCollapseParams(), 1000 / 60)
```
Comment (asvae) [inline packages/ui/src/components/va-collapse/VaCollapse.vue:212]: Why 1000/60? Very peculiar timing.

**Review by asvae — APPROVED** [review]
One comment, but otherwise mergeable :).

File: packages/ui/src/components/va-collapse/VaCollapse.vue (hunk 1 above)
Comment (m0ksem) [reply packages/ui/src/components/va-collapse/VaCollapse.vue:212]: 16ms. I thought 1000 / 60fps will be much readable than 16ms.

Comment (m0ksem) [inline packages/ui/src/components/va-collapse/VaCollapse.vue:212]:
```suggestion
      setTimeout(() => this.setCollapseParams(), 16)
```

**Pull request merged**